packages/nuxt-start/package.json

```diff
@@ -67,12 +67,12 @@
     "fresh": "^0.5.2",
     "serialize-javascript": "^1.5.0",
     "serve-static": "^1.13.2",
-    "compression": "^1.7.3",
     "connect": "^3.6.5",
     "launch-editor-middleware": "^2.2.1",
     "server-destroy": "^1.0.1",
     "chalk": "^2.4.1",
     "esm": "^3.0.80",
-    "ip": "^1.1.5"
+    "ip": "^1.1.5",
```

Comment:
Is this change necessary ? 😄

Reply:
Oh well 🙈 
Of course not!

Reply:
Maybe I missed sth, if you generated this by `yarn build`, I think it's fine.